### DISCUSSION TITLE
Restarting simulation with colvars

### DISCUSSION THREAD

**Discussion by Praveen-Prabhakar** [Colvars/colvars#657] [Q&A] [answered]
Hi, 

During my simulations, occasionally what happens is the lipid membrane comes out of the simulation box, hence I have to wrap it and realign it inside the box and restart the simulation. For wrapping and realigning I use the .restart.coor file from the NAMD restart output. And I save the wrapped & realigned molecule as a pdb and restart the simulation by providing velocity using the temperature. 

But when I restart I get the error "colvars:  differs greatly from the value last read from the state file:"
colvars: Possible causes are changes in configuration, wrong state file, or how PBC wrapping is handled

The CV value calculated from the state file was -23.04 and the value from the pdb it is -23.15. 

I am getting an error even though the difference is small. Is there any way work around this? Please let me know (Also, my restart frequencies for .coor.restart and .restart.colvars.state are same)

Please let me know if any other information is required,

Thank you, 
Praveen

**Comment by HanatoK** [comment, marked as answer]
@Praveen-2k21
Hi Praveen,

There is indeed an workaround. You can open your .restart.colvars.state by a text editor, and search "-23.04", and it should appear in a `colvar {...}` block similar to:

```
colvar {
  name distanceX
  x  2.05694280926274e+00
  extended_x  2.11022699034402e+00
  extended_v  2.25143697478233e-03
}
```
You may see `x -23.04` or something similar with a differenct precision. You can change it to `x -23.15` and try NAMD again.

**Comment by giacomofiorin** [comment]
@HanatoK is correct. Also, if you use a version more recent than February 2020 (e.g. NAMD 2.14) you can also delete that entire `colvar { ... }` block from the state file, and it should be read correctly even if it's missing.

Please reopen if either of these suggestions don't work.
